heliopy/data/util.py

```diff
@@ -238,6 +237,15 @@ def cdf_units(cdf_, manual_units=None):
                 warnings.warn(message, Warning)
         except KeyError:
             continue
+        try:
+            if len(cdf_[key][0]) > 1:
```

Comment:
What is `len(cdf_[key][0])`? Is it the number of columns in that CDF field? If so it might make the code more readable to do something like `ncols = len(cdf_[key][0])` first.

Reply:
Yep! This is to count the number of columns in a given field.